src/ApiTestCase.php

```diff
@@ -0,0 +1,202 @@
+<?php
+
+/*
+ * This file is part of the Lakion package.
+ *
+ * (c) Lakion
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Lakion\ApiTestCase;
+
+use Coduo\PHPMatcher\Factory\SimpleFactory;
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * @author Łukasz Chruściel <lukasz.chrusciel@lakion.com>
+ * @author Paweł Jędrzęjewski <pawel.jedrzejewski@lakion.com>
+ * @author Michał Marcinkowski <michal.marcinkowski@lakion.com>
+ */
+abstract class ApiTestCase extends WebTestCase
+{
+    /**
+     * @var \Symfony\Bundle\FrameworkBundle\Client
+     */
+    protected $client;
+
+    public function setUp()
+    {
+        $this->client = static::createClient(array(), array('HTTP_Accept' => 'application/json'));
+    }
+
+    public function tearDown()
+    {
+        foreach ($this->client->getContainer()->getMockedServices() as $id => $service) {
+            $this->client->getContainer()->unmock($id);
+        }
+
+        \Mockery::close();
+        $this->client = null;
+
+        parent::tearDown();
+    }
+
+    /**
+     * Gets service from DIC.
+     *
+     * @param string $id
+     *
+     * @return object
+     */
+    protected function get($id)
+    {
+        return $this->client->getContainer()->get($id);
+    }
+
+    /**
+     * @param Response $response
+     * @param int      $statusCode
+     */
+    protected function assertResponseCode(Response $response, $statusCode)
+    {
+        $this->assertEquals($statusCode, $response->getStatusCode());
+    }
+
+    /**
+     * @param Response $response
+     *
+     * @throws \Exception
+     */
+    protected function assertSuccessfulResponse(Response $response)
```

Comment:
I think that this should not be a separate method. I'd add it to ``assertJsonResponseContent``.